src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -562,9 +562,11 @@
     </ItemGroup>
   </Target>
 
-  <!-- Native AOT properties should be shown by default, except for WPF, WinForms, and class library projects. -->
+  <!-- Native AOT properties should be shown by default for projects targeting .NET 8 or higher, except for WPF and WinForms projects -->
   <PropertyGroup>
-    <ShowNativeAOTProperties Condition="'$(ShowNativeAOTProperties)' == '' and '$(UseWPF)' == 'false' and '$(UseWindowsForms)' == 'false' and $(OutputType) != 'Library'">true</ShowNativeAOTProperties>
+    <ShowNativeAOTProperties Condition="'$(ShowNativeAOTProperties)' == ''
```

Comment:
The downside to splitting the check in two is that the existing value of `ShowNativeAOTProperties` will be ignored if `UseWPF` or `UseWindowsForms` is set.

Reply:
that's true; AFAIK there's no planned support for WPF nor WinForms, so maybe it's okay if it gets ignored for now.